ani-cli

```diff
@@ -370,6 +386,9 @@ done
 [ "$use_external_menu" = "1" ] && multi_selection_flag="${ANI_CLI_MULTI_SELECTION:-"-multi-select"}"
 printf "\33[2K\r\033[1;34mChecking dependencies...\033\n[0m"
 dep_ch "curl" "sed" "grep" || true
+if [ "$skip_intro" = 1 ]; then
+    dep_ch "ani-skip" || true
+fi
```

Comment:
could this be consolidated to one line or does shellecheck complain then?

Reply:
It did complain, but I realised I could  put the second statement into brackets – so it's done.

Reply:
Can this be resolved?